docs/swagger/api.yml

```diff
@@ -0,0 +1,620 @@
+openapi: 3.0.0
+# Added by API Auto Mocking Plugin
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/sidetree/0.0.1
+info:
+  description: This is an experimental Open API Specification for the Sidetree Protocol.
+  version: "0.0.1"
+  title: Sidetree REST API
+tags:
+  - name: Node
```

Comment:
@OR13 Where are the DID Method operations going?
https://github.com/decentralized-identity/sidetree/blob/master/docs/protocol.md#sidetree-rest-api

Reply:
I think they are entirely missing from this.

Reply:
@OR13 awesome. thanks!

Reply:
I added some endpoints, but IMO we need to define the JWS in the Core Spec, not in the API interface (compact JWS is normative)